unix/xserver/hw/vnc/xvnc.c

```diff
@@ -766,10 +766,11 @@ vfbUninstallColormap(ColormapPtr pmap)
 	    curpmap = (ColormapPtr) LookupIDByType(pmap->pScreen->defColormap,
 						   RT_COLORMAP);
 #else
-	    dixLookupResourceByType((void * *) &curpmap, pmap->pScreen->defColormap,
-				    RT_COLORMAP, serverClient, DixUnknownAccess);
+	    int rc =  dixLookupResourceByType((void * *) &curpmap, pmap->pScreen->defColormap,
+					      RT_COLORMAP, serverClient, DixUnknownAccess);
 #endif
-	    (*pmap->pScreen->InstallColormap)(curpmap);
+	    if (rc == Success)
```

Comment:
`rc` will be missing in the other `#if` code path. Better to check it early and return if it is a failure (and log).